contracts/SplitEquippable/README.md

```diff
@@ -0,0 +1,1348 @@
+# SplitEquippable
+
+The `ExternalEquippable` composite of RMRK legos uses the [`Nesting`](../Nesting/README.md),
+[`MultiResource`](../MultiResource/README.md), [`Equippable`](../MergedEquippable/README.md#equippable) and
+[`Base`](../MergedEquippable/README.md#base) RMRK legos. Unlike [`SplitEquippable`](../SplitEquippable/README.md) RMRK
```

Comment:
Maybe you meant unlike the merged version?

Reply:
Yep, I guess I was already a bit docs loopy at this point 😂